static/app/utils/gettingStartedDocs/getPlatformPath.ts

```diff
@@ -0,0 +1,40 @@
+import type {PlatformIntegration} from 'sentry/types/project';
+
+export function getPlatformPath(platform: PlatformIntegration) {
+  // TODO: This will be removed once we no longer rely on sentry-docs to load platform icons
```

Comment:
@priscilawebdev you added this todo, right?
Let's sync at one point if we are ready to get rid of this logic.

Reply:
@ArthurKnaus I have worked on it in the PR https://github.com/getsentry/sentry/pull/72764